monash-human-power/MHP-website
# Add blog

## Description



Add a system where people can create blog posts!

You can use Netlify CMS to do it (recommended) or add a makrdown file in `/src/blog`.

## Screenshots



![Screenshot 2022-12-07 at 18-53-02 hello world Monash Human Power](https://user-images.githubusercontent.com/34503494/206889080-a63394f0-8747-4046-979c-9d5976583641.png)


## Steps to Test



Try to create a post in `/src/blog` or Netlify CMS.

Until this PR is merged, all posts made through the CMS will not appear.

Note that you should NOT delete `.gitkeep.md` as it's used to prevent build errors when there is no blog posts.
